recipes/nyxus/run_test.py

```diff
@@ -0,0 +1,5 @@
+import nyxus
```

Comment:
https://docs.conda.io/projects/conda-build/en/latest/concepts/recipe.html

This file is run twice because it is named run_test.[py] and also you call it in the test/commands section. Is it even needed? The import of nyxus is also tested in test/imports. Remove this file if it is redundant.

Reply:
Thanks for the explanation. Removed the file.